packages/react-charts/src/components/ChartScatter/examples/ChartScatter.md

```diff
@@ -30,16 +30,51 @@ Note: PatternFly React charts live in its own package at [@patternfly/react-char
 PatternFly React charts are based on the [Victory](https://formidable.com/open-source/victory/docs/victory-chart/) chart library, along with additional functionality, custom components, and theming for PatternFly. This provides a collection of React based components you can use to build PatternFly patterns with consistent markup, styling, and behavior.
 
 ## Examples
-### Area chart
+### Basic
+```js
+import React from 'react';
+import { Chart, ChartAxis, ChartGroup, ChartLine, ChartScatter, ChartThemeColor, ChartVoronoiContainer, getResizeObserver } from '@patternfly/react-charts';
 
-This demonstrates how to add interactive data points to an area chart.
+<div style={{ height: '275px', width: '450px' }}>
+  <Chart
+    ariaDesc="Average number of pets"
+    ariaTitle="Scatter chart example"
+    containerComponent={
+      <ChartVoronoiContainer
```

Comment:
Is there anywhere in the chart docs that explain what the `ChartVoronoiContainer` is intended to be used for and what its props are? I noticed that the Donut and Pie charts don't use them but some of the other charts do and I cannot find a pattern for when it'd be used and why.

Reply:
I added `ChartVoronoiContainer` to this example's `propComponents`.

FYI, `ChartVoronoiContainer` adds the ability to associate a mouse position with the data point(s) closest to it. This container is useful for adding hover interactions, like tooltips. It's not necessary for pie charts because they don't have an x and y axis.

Unfortunately, the docs only include `ChartVoronoiContainerProps`, it will not include docs for `ChartVoronoiContainer` itself. Perhaps we can discuss offline how the docs gen can include this information?

Reply:
Unfortunately, the docs only include props for `ChartVoronoiContainerProps` interface, it will not include comments for `ChartVoronoiContainerProps` or any docs for `ChartVoronoiContainer` itself. ideally, these comments would appear in the examples with the props. 

Perhaps we can discuss offline how the docs gen can include this information?

Reply:
Perhaps the docs generator can be updated to include this information? I can see the comments are available in the `getInterfaceMetadata` function of `theme-patternfly-org/scripts/tsDocgen.js`.

Reply:
Created https://github.com/patternfly/patternfly-react/issues/7612, so we might include the interface documentation in the examples.

Reply:
In order to simplify this PR, I added interface descriptions via https://github.com/patternfly/patternfly-react/pull/7640.